src/ProductlineApp.WebUI/ClientApp/src/components/molecules/settingsSections/buttonsSection/ButtonsSection.tsx

```diff
@@ -0,0 +1,14 @@
+import CancelButton from '../../../atoms/buttons/cancelButton/CancelButton';
+import ConfirmChangePersonalDataButton from '../../../atoms/buttons/confirmChangePersonalDataButton/ConfirmAccountDataButton';
+import './css/settingsButtonsSection.css';
+
+function ButtonsSection({ setShowButtons, showButtons }: { setShowButtons?: any, showButtons?: boolean }) {
```

Comment:
to co poprzednio - wystarczy przekazac jedna referencje do funkcji i obsluzyc ja w komponencie nadrzednym